### ISSUE TITLE
Support darkmode / prefers-color-scheme

### ISSUE THREAD

**Issue by psychemedia** [innovationOUtside/nb_extension_empinken#8]
Browser CSS supports `prefers-color-scheme` in `light` and `dark` modes.

Empinken should be sensitive to the mode and render cells appropriately.
